ethexe/blob-loader/src/lib.rs

```diff
@@ -254,25 +259,33 @@ impl BlobLoaderService for BlobLoader {
 
     fn load_codes(&mut self, codes: HashSet<CodeId>, attempts: Option<u8>) -> Result<()> {
         for code_id in codes {
-            if self.codes_loading.contains(&code_id) || self.db.original_code_exists(code_id) {
+            if self.codes_loading.contains(&code_id) {
                 continue;
             }
 
-            let code_info = self
+            let CodeBlobInfo { timestamp, tx_hash } = self
                 .db
                 .code_blob_info(code_id)
                 .ok_or(anyhow!("not found code info for {code_id} in db"))?;
 
+            if let Some(code) = self.db.original_code(code_id) {
```

Comment:
Optimisation - if original code in db - takes it from db, but still generates output event because maybe it's needed for outer services